applications/ParticleMechanicsApplication/python_scripts/deprecated_python_scripts/deprecated_particle_mpm_solver.py

```diff
@@ -0,0 +1,370 @@
+        model_part.AddNodalSolutionStepVariable(KratosParticle.AUX_T)
+        model_part.AddNodalSolutionStepVariable(KratosParticle.AUX_R_VEL)
+        model_part.AddNodalSolutionStepVariable(KratosParticle.AUX_T_VEL)
+        model_part.AddNodalSolutionStepVariable(KratosParticle.AUX_R_ACC)
+        model_part.AddNodalSolutionStepVariable(KratosParticle.AUX_T_ACC)
+        model_part.AddNodalSolutionStepVariable(KratosMultiphysics.DENSITY)
+        model_part.AddNodalSolutionStepVariable(KratosParticle.NODAL_LUMPED_MASS)
+        
+        # Add variables for arbitrary slope with slip
+        model_part.AddNodalSolutionStepVariable(KratosMultiphysics.IS_STRUCTURE)
+        model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NORMAL)
+        
+        # Add variables for specific cases
+        if self.settings["rotation_dofs"].GetBool():
+            # add specific variables for the problem (rotation dofs)
+            model_part.AddNodalSolutionStepVariable(KratosMultiphysics.ROTATION)
+            model_part.AddNodalSolutionStepVariable(KratosMultiphysics.TORQUE)
+            model_part.AddNodalSolutionStepVariable(KratosMultiphysics.ANGULAR_VELOCITY)
+            model_part.AddNodalSolutionStepVariable(KratosMultiphysics.ANGULAR_ACCELERATION)
+        if self.settings["pressure_dofs"].GetBool():
+            # add specific variables for the problem (pressure dofs)
+            model_part.AddNodalSolutionStepVariable(KratosSolid.PRESSURE_REACTION)
+            model_part.AddNodalSolutionStepVariable(KratosParticle.NODAL_MPRESSURE)
+            model_part.AddNodalSolutionStepVariable(KratosParticle.AUX_PRESSURE)
+        if self.settings["time_integration_method"].GetString() == "explicit" :
+            model_part.AddNodalSolutionStepVariable(KratosMultiphysics.FORCE_RESIDUAL)
+            model_part.AddNodalSolutionStepVariable(KratosMultiphysics.MIDDLE_VELOCITY)
+
+    def _model_part_reading(self):
+        # reading the model part of the background grid
+        if(self.settings["grid_model_import_settings"]["input_type"].GetString() == "mdpa"):
+            KratosMultiphysics.ModelPartIO(self.settings["grid_model_import_settings"]["input_filename"].GetString()).ReadModelPart(self.model_part1)
+        else:
+            raise Exception("Other input options are not implemented yet.")
+        
+        # reading the model part of the material point
+        if(self.settings["model_import_settings"]["input_type"].GetString() == "mdpa"):
+            KratosMultiphysics.ModelPartIO(self.settings["model_import_settings"]["input_filename"].GetString()).ReadModelPart(self.model_part2)
+        else:
+            raise Exception("Other input options are not implemented yet.")
+
+    def _execute_check_and_prepare(self):
+        # Specific active node and element check for particle MPM solver
+        for node in self.model_part1.Nodes:
+            if (node.Is(KratosMultiphysics.ACTIVE)):
+                print(node.Id)
```

Comment:
Use the logger, besides, this is a lot of prints

Reply:
this is in the deprecated solver, @bodhinandach wont touch this